docs/_sass/components/_masthead.scss

```diff
@@ -10,6 +10,20 @@
       $brand-secondary
     );
 
+  .masthead-background {
+  	background: url("../../../img/logo_masthead.svg");
+  	position: absolute;
+  	top: 0;
+  	left: 0;
+  	width: 100%;
+  	height: 100%;
+  	background-position: -500px -1350px;
```

Comment:
could it be a problem in devices with a different ratio, for example an iPhone 5? (just asking)

Reply:
It shouldn't be specially affected, I think.

iPhone SE - Actual device (DPR 2)

![photo_2019-05-07_14-57-27](https://user-images.githubusercontent.com/7753447/57301084-733af000-70d8-11e9-8e08-53eadc3d68a4.jpg)

iPhone SE - Browser emulated (DPR 2)

![Captura de pantalla de 2019-05-07 14-54-59](https://user-images.githubusercontent.com/7753447/57301135-8ea5fb00-70d8-11e9-8ff6-26b9104e7be1.png)

Galaxy S9 - Browser emulated (DPR 4)

![Screen Shot 2019-05-07 at 14 56 49](https://user-images.githubusercontent.com/7753447/57301175-a9786f80-70d8-11e9-9332-5f748d3d3c0a.png)

Reply:
It looks nice! Good job! :)